settings.py

```diff
@@ -1179,7 +1184,7 @@ def get_user_url(user):
         'AUDIENCE': 'https://developer.mozilla.org',
         'REQUEST_PARAMETERS': {
             'siteName': 'Mozilla Developer Network',
-            'siteLogo': MEDIA_URL + 'img/opengraph-logo.png',
+            'siteLogo': STATIC_URL + 'img/opengraph-logo.png',
```

Comment:
Sadly that's not something we can easily make it use the static tag as there is a circular import issue preventing this (we can't import static to use it here because that loads the settings, which imports static, which imports the settings etc). There is a place in allauth where those values are actually passed along to the Persona login template, so I think we can inject it there properly with the static tag, but I'm afraid we have to live with this one direct `STATIC_URL` use. I think filing a ticket would make sense for that case.